core/trino-spi/src/main/java/io/trino/spi/connector/ColumnHandle.java

```diff
@@ -13,6 +13,21 @@
  */
 package io.trino.spi.connector;
 
+/**
+ * A handle to a column returned from the connector to the engine.
```

Comment:
Maybe "Represents a column within a table".

The "within" would also hint at the equals contract.